example/World.tsx

```diff
@@ -0,0 +1,31 @@
+import Vue ,{ CreateElement } from 'vue'
+import Component from '../lib/index'
+
+@Component({
+  props:{
+    title:{
+      type:String,
+      required:true
+    }
+  }
+})
+export default class World extends Vue {
+  title:string
+  content:string = 'default content'
+  get computedContent () {
+    return 'computed ' + this.content
+  }
+
+  clearContent() {
+    this.content = ''
+  }
+  render(h: CreateElement) {
+    return <div >
+      <h1>{this.title}</h1>
```

Comment:
Can we simplify the render function and the component definition more? I think it could have only one simple element since we just want to show tsx is available in this component. `props`/`methods`/`computed` shouldn't be there since they just complicate the example, IMO.